libs/tools/utils/publish.js

```diff
@@ -5,7 +5,7 @@ const userCanPublishPage = async (detail, isBulk = true) => {
   const { live, profile, webPath } = detail;
   let canPublish = isBulk ? live?.permissions?.includes('write') : true;
   let message = 'Publishing is currently disabled for this page';
-  const config = await getCustomConfig('/.milo/publish-permissions-config.json');
+  const config = await getCustomConfig('/.milo/publish-permissions-config.json?limit=50000');
```

Comment:
@narcis-radu I would guess you need to update the URL based on which this is being mocked 🤔 https://github.com/adobecom/milo/blob/stage/test/blocks/bulk-publish-v2/mocks/fetch.js#L15